builtin-functions/_functions.txt

```diff
@@ -267,6 +267,7 @@ function array_slice ($a ::: array, $offset ::: int, $length = null, $preserve_k
 /** @kphp-extern-func-info cpp_template_call */
 function array_pad ($a ::: array, $size ::: int, $value ::: any) ::: array< ^1[*] | ^3 >;
 function array_column ($a ::: array, $column_key, $index_key = null) ::: array< ^1[*][*] > | false;
+function array_unset (&$a ::: array, any $key) ::: ^1[*];
```

Comment:
I think `mixed` type for `$key` fits more.  
```suggestion
function array_unset (&$a ::: array, mixed $key) ::: ^1[*];
```
Now you can pass class instance to `$key`, that will lead to gcc error

Reply:
Discussed offline, I dislike to confuse user that key is mixed. Actually key maybe have another type like int, string. So better leave as is